### PR TITLE
Feature/130 impl new master actor

### PR DESCRIPTION
**Pull request by ztkmkoo**
Ref #130 

I'm working on re-design actor system.
I will resolve the issue in two pull requests(contains this one).
And it is the first step,

- design master actor interface

### CODE REVIEW

**Comment by ztkmkoo** [comment]
@Alencion @Doyuni @RulLu16 @yeonnseok 
Feel free to review this, if you have any advice.
(til 2020.09.19)

Thx

File: dss-core/src/main/java/io/github/ztkmkoo/dss/core/actor/DssServiceSpawnable.java [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    <P extends DssServiceActorProperty> DssServiceActorResolvable<String> createDssServiceActorResolvable(P property, ActorRef<DssServiceCommand> actor);
+
+    /**
+     * spawn service actors and set it
+     */
+    default <M extends DssMasterActorProperty> void initializeServiceActor(AbstractDssActor<DssMasterCommand> master, M masterProperty) {
+        final List<DssServiceActorProperty> propertyList = createDssServiceActorPropertyList(masterProperty);
+        if (Objects.isNull(propertyList) || propertyList.isEmpty()) {
+            return;
+        }
```
Comment (RulLu16) [inline dss-core/src/main/java/io/github/ztkmkoo/dss/core/actor/DssServiceSpawnable.java]: Hi, I just wonder that propertyList.isEmpty() is necessary. 
I think createDssServiceActorPropertyList can return null when there are no property.

Or is this empty propertyList used in other parts?

Comment (Doyuni) [reply dss-core/src/main/java/io/github/ztkmkoo/dss/core/actor/DssServiceSpawnable.java]: I think that it consider this case.
`createDssServiceActorPropertyList` can return null or empty list.
For example 
```java
final List<Integer> list = new ArrayList<>();

if (list.isEmpty()) {
    // list is empty and not null
}
if (Objects.isNull(list)) { }
```

```java
final List<Integer> list = null;

if (Objects.isNull(list)) {
    // list is null
}
/*
if (list.isEmpty()) { }
*/
```

**Review by Doyuni — APPROVED** [review]

File: dss-core/src/main/java/io/github/ztkmkoo/dss/core/actor/DssServiceSpawnable.java (hunk 1 above)
Comment (ztkmkoo) [reply dss-core/src/main/java/io/github/ztkmkoo/dss/core/actor/DssServiceSpawnable.java]: @RulLu16 
Hi there,
I check null and empty for @Doyuni pointed out.
(I did it automatically..)

But, as you said, it is not neccessary to check empty, if using Collection stream.
I will modify this

**Review by Doyuni — APPROVED** [review]

**Review by RulLu16 — APPROVED** [review]

**Pull request merged**